src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/MarkDuplicatesSparkIntegrationTest.java

```diff
@@ -189,6 +189,14 @@ public void testSupplementaryReadUnmappedMate() {
         }
     }
 
+    @Test
+    public void test() {
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addOutput(createTempFile("output","bam"));
+        args.addInput(new File("/Users/emeryj/hellbender/gatk/compareDuplicates/picarddiffs.sam"));
```

Comment:
This path is local to your machine.

Reply:
whoops, that was a test file from my evaluation, ill get rid of that